tests/test_simresults.py

```diff
@@ -277,7 +278,7 @@ def test_expect_noisy(results_noisy):
     with pytest.raises(ValueError, match="non-diagonal"):
         results_noisy.expect([bad_op])
     op = qutip.tensor([qutip.qeye(2), qutip.basis(2, 0).proj()])
-    assert np.isclose(results_noisy.expect([op])[0][-1], 0.7466666666666666)
+    assert np.isclose(results_noisy.expect([op])[0][-1], 0.7333333333333334)
```

Comment:
Do you understand why this changed?

Reply:
Yes, because the `results_noisy` fixture changed (see the assert I have added). The value of 'amp_sigma' was not changed before when we were doing the "add_config". I thought that fix would not change anything but turns out we lived with this mistake all along ^^'

Reply:
Jeez... Good catch!